infrastructure/prod.tfvars

```diff
@@ -1,2 +1,4 @@
 idam_api_base_uri = "https://prod-idamapi.reform.hmcts.net:3511"
 dns_server="10.96.75.254"
+enable_form_definition_endpoint="false"
```

Comment:
Will we not need that in production?

Reply:
not for now, as no service is going to be using it